scripts/run-integration-tests-source.sh

```diff
@@ -167,7 +167,7 @@ function setup_node() {
   # Set NODE_ENV=development to rebuild hopr-admin next files
   # at runtime. Necessary to start multiple instances of hoprd
   # in parallel
-  DEBUG="hopr*" NODE_ENV=development node packages/hoprd/lib/index.js \
```

Comment:
There are more occurrences of this in `scripts/` which need to be adapted.

Reply:
Changed